source/_docs/installation.markdown

```diff
@@ -106,10 +106,4 @@ These guides are provided as-is. Some of these install methods are more limited
     </div>
     <div class='title'>Synology</div>
   </a>
-  <a class='option-card' href='/docs/installation/freenas/'>
-    <div class='img-container'>
-      <img src='/images/supported_brands/freenas.png' />
-    </div>
-    <div class='title'>FreeNAS</div>
-  </a>
```

Comment:
If you want to make an installation of Home Assistant then you will look in the documentation and not the forum. 

The section which contains that guide was renamed to "Community provided guides" at some point but actually they are OS-specify guide. That they are not "official" but definitely part of "Home Assistant provides multiple ways to be installed." and we don't want you to re-invent the wheel again especially as using a `venv` is mentioned as a way to run HA.
.

Reply:
It is being removed.